File: Client/src/app/models/Organization.ts
export class OrganizationTableData {
    rowNum?: string;
    groupName?: string;
    totalInstitute?: string;
    instructorCount?: string;
    paidStudentCount?: string;
    trialStudentCount?: string;
  }

  export class OrganizationTableHeader{
    srNo?: string;
    groupConfigLabel?: string;
    instituteLabelConfig?: string;
    instructorLabelConfig?: string;
    subScribeStudentLabelConfig?: string;
    trialStudentLabelConfig?: string;
  }

  export class OrganizationDropDownData {
    groupId?: number;
    groupName?: string;
    collegeId?: number;
    showNewUserTab?: string;
    showAssessmentsTab?: string;
    masterCourseId?: number;
    assessmentTabName?: string;
    organizationType?: string;
    organizationId?: number;
  }

File: API/CMAdmin.API/Models/Organization.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CMAdmin.API.Models
{
    public class Organization
    {
        public int GroupId { get; set; }
        public string GroupName { get; set; }
        public int CollegeId { get; set; }
        public string ShowNewUserTab { get; set; }
        public string ShowAssessmentsTab { get; set; }
        public int MasterCourseId { get; set; }
        public string AssessmentTabName { get; set; }
        public string OrganizationType { get; set; }
        public int OrganizationId { get; set; }
    }

    public class CreateOrganization
    {
        public int GroupId { get; set; }
        public string GroupName { get; set; }
        public string OrganizationType { get; set; }
    }
}
